BroadcomCloudSwg/connectors/metrics.py

```diff
@@ -0,0 +1,25 @@
+from prometheus_client import Counter, Histogram
+
+# Declare prometheus metrics
+prom_namespace = "symphony_module_broadcom_cloud_swg"
```

Comment:
For the three following metrics, the namespace must be the common one
```suggestion
prom_namespace = "symphony_module_common"
```